app/core/store/modules/levelSessions.js

```diff
@@ -0,0 +1,118 @@
+import levelSessionsApi from 'core/api/level-sessions'
+
+const SESSIONS_PER_REQUEST = 10
+
+export default {
+  namespaced: true,
+
+  state: {
+    loading: {
+      sessionsByClassroom: {}
+    },
+
+    // CLASSROOM_ID: {
+    //   sessions: [],
+    //   levelCompletionsByUser: {
+    //      LEVEL_ID: true / false
+    //    }
+    // }
```

Comment:
We can remove these comments if not relevant

Reply:
Updated this comment to make it more clear - let me know what you think.

Reply:
Yep looks clear now. Thanks.